platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -4632,9 +4632,7 @@
 
             if (!container.is(e.target) && container.has(e.target).length === 0 &&
                 !button.is(e.target) && button.has(e.target).length === 0 && container.is(":visible")) {
-                if (e.target.offsetParent.hasAttribute("id") && e.target.offsetParent.id !== "ui-datepicker-div") {
-                    $scope.showLookupDialog();
-                }
```

Comment:
You can not remove this 'if' condition, because this condition prevents closing lookups panel when user operate on datepicker and also when user wants to close the datepicker panel by clicking outside of this panel.